pkg/3scale/amp/component/system_options_builder.go

```diff
@@ -13,6 +13,8 @@ type S3FileStorageOptions struct {
 	AWSRegion            string
 	AWSBucket            string
 	AWSCredentialsSecret string
+	EndpointScheme       string
+	EndpointHost         string
```

Comment:
TODO:
In dev environments the port is often set to a non-standard port.
Talk with system team on whether we can set the port or not. I've assumed  that we can and the way to provide it is via the EndpointHost field (separated by a colon when you want to specify a port). In that case maybe a better name than EndpointHost could be used (as it optionally  considers that an optional port separated by a colon can be provided in this attribute)

Reply:
I also thought on use a simple Endpoint attribute that included scheme, host and port but System team gave me feedback saying it would require parsing on their side and that they prefer to split it. Talk with them about that if you consider we should unify everything